docs/source/using/addons.adoc

```diff
@@ -267,6 +267,23 @@ $ minishift addons remove admin-user
 admin user deleted
 ----
 
+[[uninstall-addons]]
+== Uninstalling Add-ons
+
+Add-ons can be uninstall with the xref:../command-ref/minishift_addons_uninstall.adoc#[`minishift addons uninstall`] command.
+It is the mirror command to xref:../command-ref/minishift_addons_install.adoc#[`minishift addons install`] command and can be used regardless whether the add-on is enabled or not.
+Provided it exists, `minishift addons uninstall` will delete add-on specific directory.
```

Comment:
"Provided it exists, `minishift addons uninstall` will delete the whole add-on directory."